src/pageObject.js

```diff
@@ -1,15 +1,33 @@
 'use strict';
 
 class Header {
-
+  constructor() {
+    this.logo = 'Conduit logo';
+  }
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
-
+  constructor(url, articleId) {
+    super(url);
+    this.articleId = articleId;
+    this.commentButton = '[Publish comment] button';
```

Comment:
```suggestion
    this.commentButton = '[Publish comment] button';
```
this should be the getter property